.github/workflows/prerelease.yml

```diff
@@ -142,11 +122,11 @@ jobs:
       - name: Download zip from GH Release assets and extract .exe
         shell: pwsh
         run: |
-          build\windows\download_zip_extract_exe.ps1 "$env:INTEGRATION" ${{ matrix.goarch }} "$env:TAG" "$env:REPO_FULL_NAME"
+          build\windows\download_zip.ps1 "$env:INTEGRATION" ${{ matrix.goarch }} "$env:TAG" "$env:REPO_FULL_NAME"
+          build\windows\extract_exe.ps1 "$env:INTEGRATION" ${{ matrix.goarch }} "$env:TAG"
```

Comment:
The same that the previous comment in the `push_pr` workflow.

Reply:
Let's keep the conversation in the [same thread](https://github.com/newrelic/nri-mssql/pull/108#discussion_r1136815893). I am marking this as resolved.